packages/extra/bootbox/templates/matchbox/deployment.yaml

```
@@ -0,0 +1,54 @@
+        args:
+          - "-address=:8080"
+          - "-log-level=debug"
+        volumeMounts:
+          - name: profiles-volume
+            mountPath: /var/lib/matchbox/profiles
+          - name: groups-volume
+            mountPath: /var/lib/matchbox/groups
+          - name: configs-volume
+            mountPath: /var/lib/matchbox/assets/configs
+        ports:
+          - name: http
```

yamllint (1.35.1)

[warning] wrong indentation: expected 8 but found 10

(indentation)